execution_engine_testing/tests/src/test/regression/ee_599.rs

```diff
@@ -9,7 +9,7 @@ use casper_types::{account::AccountHash, runtime_args, RuntimeArgs, U512};
 const CONTRACT_EE_599_REGRESSION: &str = "ee_599_regression.wasm";
 const CONTRACT_TRANSFER_TO_ACCOUNT: &str = "transfer_to_account_u512.wasm";
 const DONATION_PURSE_COPY_KEY: &str = "donation_purse_copy";
-const EXPECTED_ERROR: &str = "InvalidContext";
+const EXPECTED_ERROR: &str = "ForgedReference";
```

Comment:
presumably this is because there is no longer a distinction btwn account and contract contexts; its all just addressable entity contexts...thus the true error is forged uref?